test/api/v1/guestbook_types.go

```diff
@@ -35,7 +35,7 @@ type GuestbookSpec struct {
 
 // GuestbookEntry defines an entry in a guest book.
 type GuestbookEntry struct {
-	// Name of the guest
+	// Name | of the guest
```

Comment:
Something like that would be clearer on the intentions?
```suggestion
	// Name of the guest (pipe | should be escaped)
```

Reply:
Resolved in 4d6257f1f759972f7d7650c38b7891cee3a3df23